Install fails on arch

Trying to reinstall Linuwu-Sense. Ran make uninstall, reinstalled linux-headers, cloned the repo, ran make install, ending up with this:

`make -C /lib/modules/6.19.6-arch1-1/build M=/home/tech/Linuwu-Sense modules`
`make[1]: Entering directory '/usr/lib/modules/6.19.6-arch1-1/build'`
`make[2]: Entering directory '/home/tech/Linuwu-Sense'`
`make[2]: Leaving directory '/home/tech/Linuwu-Sense'`
`make[1]: Leaving directory '/usr/lib/modules/6.19.6-arch1-1/build'`
`# --- auto sign block ---`
`#finding sign-file tool`
`if [ -x "/lib/modules/6.19.6-arch1-1/build/scripts/sign-file" ]; then \`
`	SIGN_TOOL="/lib/modules/6.19.6-arch1-1/build/scripts/sign-file"; \`
`elif [ -x "/usr/src/linux-headers-6.19.6-arch1-1/scripts/sign-file" ]; then \`
`	SIGN_TOOL="/usr/src/linux-headers-6.19.6-arch1-1/scripts/sign-file"; \`
`else \`
`	echo "ERROR: sign-file tool not found"; \`
`	exit 1; \`
`fi; \`

`# assuming keys are located in a ~/module-signing folder named MOK....`
`echo "Signing module linuwu_sense.ko using $SIGN_TOOL"; \`
`sudo $SIGN_TOOL sha256 \`
`	/home/tech/module-signing/MOK.priv \  `
`Signing module linuwu_sense.ko using `
`[sudo] password for tech: `
`sudo: sha256: command not found`
`make: *** [Makefile:18: all] Error 1`

--- a/Makefile
+++ b/Makefile
@@ -12,23 +12,24 @@ all:
 	$(MAKE) -C $(KDIR) M=$(PWD) modules
 
 	# --- auto sign block ---
-	
-	#finding sign-file tool
+	# Check if keys exist before attempting to sign
+	@if [ -f "$(HOME)/module-signing/MOK.priv" ] && [ -f "$(HOME)/module-signing/MOK.der" ]; then \
 	if [ -x "/lib/modules/$(KVER)/build/scripts/sign-file" ]; then \
-		SIGN_TOOL="/lib/modules/$(KVER)/build/scripts/sign-file"; \
+	SIGN_TOOL="/lib/modules/$(KVER)/build/scripts/sign-file"; \
 	elif [ -x "/usr/src/linux-headers-$(KVER)/scripts/sign-file" ]; then \
-		SIGN_TOOL="/usr/src/linux-headers-$(KVER)/scripts/sign-file"; \
+	SIGN_TOOL="/usr/src/linux-headers-$(KVER)/scripts/sign-file"; \
 	else \
-		echo "ERROR: sign-file tool not found"; \
-		exit 1; \
+	echo "ERROR: sign-file tool not found, but MOK keys exist."; \
+	exit 1; \
 	fi; \
-
-	# assuming keys are located in a ~/module-signing folder named MOK....
 	echo "Signing module linuwu_sense.ko using $$SIGN_TOOL"; \
 	sudo $$SIGN_TOOL sha256 \
-		$(HOME)/module-signing/MOK.priv \  
-		$(HOME)/module-signing/MOK.der \
-		$(PWD)/src/linuwu_sense.ko
+	$(HOME)/module-signing/MOK.priv \
+	$(HOME)/module-signing/MOK.der \
+	$(PWD)/src/linuwu_sense.ko; \
+	else \
+	echo "MOK keys not found in ~/module-signing/. Skipping module signing (Common for non-Secure Boot)."; \
+	fi
 	# --- end auto sign block ---
 
 clean:

